src/Illuminate/Support/Collection.php

```diff
@@ -300,6 +300,27 @@ public function flip()
     }
 
     /**
+     * Determine if the callable is true for all items in the collection.
+     *
+     * @param  callable $callable
+     *
+     * @return bool
+     */
+    public function forAll(callable $callable)
+    {
+        $values = $this->values()->toArray();
+
+        while (! empty($values)) {
+            $item = array_shift($values);
+            if (! $callable($item)) {
+                return false;
+            }
+        }
+
+        return true;
+    }
```

Comment:
`$callable` is kind of a generic name for a predicate. We're using `$callback` everywhere else.